src/server/models/Organization.ts

```diff
@@ -10,6 +10,7 @@ const OrganizationSchema = new Schema(
     softDelete: {
       type: Boolean,
       required: true,
+      default: false,
```

Comment:
why is this given a default value?

Reply:
I thought the default soft delete setting for an organization should be set as false. Andrew and I talked about this as well.